### PR TITLE
Documented "appWaitDuration" capability

### PR DESCRIPTION
**Pull request by youngrafi**
## Proposed changes

This is required for appium/appium#780
Following appium/appium-android-driver/pull/178 added documentation for added capability
## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
- [x] I have read the [CONTRIBUTING](/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://docs.google.com/forms/d/1lOfXRw_0VCk7gYzjj4WLetGu7yelDVo5LWh0z7pGftE/viewform)
- [x] Lint and unit tests pass locally with my changes
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules
## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by triager** [comment]
Can one of the admins verify this patch?

File: docs/en/writing-running-appium/caps.md [hunk 1]
```diff
@@ -29,6 +29,7 @@
 |`appPackage`| Java package of the Android app you want to run|`com.example.android.myApp`, `com.android.settings`|
 |`appWaitActivity`| Activity name for the Android activity you want to wait for|`SplashActivity`|
 |`appWaitPackage`| Java package of the Android app you want to wait for|`com.example.android.myApp`, `com.android.settings`|
+|`appWaitDuration`| Timeout in milliseconds used to wait for the appWaitActivity to launch (default `20000`), `30000`|
```
Comment (imurchie) [inline docs/en/writing-running-appium/caps.md]: This needs a `|` instead of `,` before `3000`. Otherwise it gets in the wrong column in the table.

**Comment by imurchie** [comment]
Thanks for this! One small tweak, then good to go!

**Comment by triager** [comment]
Can one of the admins verify this patch?

**Comment by youngrafi** [comment]
Changed `,` to `|`

**Comment by imurchie** [comment]
Looks good! Can you squash into a single commit, and we will get it in?

Thanks!

**Comment by youngrafi** [comment]
Done

**Comment by imurchie** [comment]
Excellent! Thanks for this!

**Pull request merged**